lib/ansible/executor/task_executor.py

```diff
@@ -13,6 +13,7 @@
 import sys
 import termios
 import traceback
+from typing import List, Tuple, Union
```

Comment:
We haven't defined how we want to do type annotations as of yet in the controller code base.  As such, I think we should remove it from this PR.

Reply:
I've removed the typehinting from the core and shuffled it into the same format I saw elsewhere. 

I don't see an open issue in ansible-community/community-topics about type hints, should I create one?

Reply:
since there are many competing standards we decided to stop all type hinting being added until we choose one, then only that one will be allowed for core code.

Reply:
yep. this thread is why I created [the item in ansible/community](https://github.com/ansible/community/issues/635#issuecomment-966636254)